tests/components/detector.spec.ts

```diff
@@ -145,24 +192,39 @@ describe('Detector', () => {
           expect(detector._previewStreamStarted).to.equals(false);
           expect(detector._detectorInitialized).to.equals(false);
         });
+        it('should send detector/stop command if camera does not have the people_count command', async () => {
+          detector._usePeopleCount = false;
+          await detector.destroy();
+          expect(transportWriteStub.getCall(0).args[0]).to.equals('detector/stop');
+        });
       });
       describe('on DOWS set', () => {
-        it('should only teardown detection and framing listeners/subscribers', async () => {
+        it('should teardown detection and framing listeners/subscribers and send people_count/stop', async () => {
           detector = new Detector(deviceManager, createDummyLogger(), { DOWS: true });
-          const teardownDetectorSubscriptionSpy = sinon.spy(detector, 'teardownDetectorSubscriptions');
+          detector._usePeopleCount = true;
+          teardownDetectorSubscriptionSpy = sinon.spy(detector, 'teardownDetectorSubscriptions');
+          transportWriteStub.resolves();
           detector._detectorInitialized = true;
           await detector.destroy();
+          expect(transportWriteStub.called).to.equals(true);
+          expect(transportWriteStub.getCall(0).args[0]).to.equals('people_count/stop');
           expect(teardownDetectorSubscriptionSpy.callCount).to.equals(1);
           expect(teardownDetectorSubscriptionSpy.getCall(0).args[0]).to.undefined;
           expect(detector._detectorInitialized).to.equals(false);
         });
+        it('should not send the people_count/stop command if camera software is older than a given threshold', async () => {
+          detector = new Detector(deviceManager, createDummyLogger(), { DOWS: true });
+          transportWriteStub.resolves();
+          await detector.destroy();
+          expect(transportWriteStub.called).to.equals(false);
```

Comment:
There is no assertions that it actually sends ```people_count/stop```

Reply:
Should only assert that it sends nothing

Reply:
Should only assert that nothing is sent